app/views/api/v3/analytics/user_analytics/show.html.erb

```diff
@@ -14,178 +14,184 @@
 
 <body>
 <div class="progress-contents">
-    <div class="updated-date">Updated: <span><%= @statistics.dig(:metadata, :last_updated_at) %></span></div>
-
-    <div class="card" style="min-height: 210px;">
-      <!-- daily card switcher -->
-      <div class="next-prev">
-        <button onclick="nextSlide(+1)" class="button-next">
-          <span><%= inline_svg('icon_arrow_back.svg') %></span>
-        </button>
-        <button onclick="nextSlide(-1)" class="button-prev">
-          <span><%= inline_svg('icon_arrow_forward.svg') %></span>
-        </button>
-      </div>
+  <div class="updated-date">Updated: <span><%= @statistics.dig(:metadata, :last_updated_at) %></span></div>
+
+  <div class="card" style="min-height: 210px;">
+    <!-- daily card switcher -->
+    <div class="next-prev">
+      <button onclick="nextSlide(+1)" class="button-next">
+        <span><%= inline_svg('icon_arrow_back.svg') %></span>
+      </button>
+      <button onclick="nextSlide(-1)" class="button-prev">
+        <span><%= inline_svg('icon_arrow_forward.svg') %></span>
+      </button>
+    </div>
 
-      <!-- dump all statistics in a hidden div so that we can parse it in JS -->
-      <%= content_tag :div, id: "statistics", data: { statistics: @statistics }, style: "display: none" do %>
-      <% end %>
+    <!-- dump all statistics in a hidden div so that we can parse it in JS -->
+    <%= content_tag :div, id: "statistics", data: { statistics: @statistics }, style: "display: none" do %>
```

Comment:
whoa, I see this was here already...but how much data can this be? how big will the page size get?

Reply:
It shouldn't be that big and would remain fairly constant, since the keys are the only real data. The data primarily has a bunch of dates and counts.

I did a quick check and it should almost always roughly be in the range of ~2kb.

Reply:
This page is rendered as a webview and has a bunch of JS that operates on the data. So it's important that it can work offline.